### DISCUSSION TITLE
Beginner question: any way to configure LIFX RGB bulbs (A60)?

### DISCUSSION THREAD

**Discussion by Vedemin** [awawa-dev/HyperHDR#922] [Q&A]
Hello, sorry for a newbie question but I'm just starting. I have two A60 bulbs from LIFX and want to hook them up to the screen grabber or audio control. In the future I would also like to add LED strips to the setup but for now the LIFX are all I have. Is it possible to use HyperHDR to control these lights? How can I do it?

Thanks in advance!

**Comment by awawa-dev** [comment]
Hi
Sorry, but this bulb (A60 = A19?) is not supported, although the API with UDP support looks promising https://lan.developer.lifx.com/docs/packet-contents

**Reply by Vedemin** [reply]
I understand, sad to hear that :( Do you think the support for it could soon come to HyperHDR or do you say that it would be possible to write my own code to control it by HyperHDR? Sorry for not understanding clearly, as I said I have exactly zero previous experience with any LED systems.

**Reply by awawa-dev** [reply]
Without the device it would be very hard for me to implement it in this case. And this is the main obstacle but if you have these lamps you can try adding a new driver.

The example of the UDP device driver (udpraw used with WLED):
https://github.com/awawa-dev/HyperHDR/blob/master/include/led-drivers/net/DriverNetUdpRaw.h
https://github.com/awawa-dev/HyperHDR/blob/master/sources/led-drivers/net/DriverNetUdpRaw.cpp
and its configuration scheme:
https://github.com/awawa-dev/HyperHDR/blob/master/sources/led-drivers/schemas/schema-udpraw.json
new scheme needs to be added to:
https://github.com/awawa-dev/HyperHDR/blob/master/sources/led-drivers/LedDeviceSchemas.qrc

**Reply by awawa-dev** [reply]
Should be supported now:  https://github.com/awawa-dev/HyperHDR/actions/runs/18266108329
